[2023] configure first says that it found Python, and then fails because it says that it didn't find Python

```
-- The CXX compiler identification is Clang 16.0.6
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/local/libexec/ccache/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Warning at /usr/local/share/cmake/Modules/FindBoost.cmake:1390 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/local/share/cmake/Modules/FindBoost.cmake:1514 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/local/share/cmake/Modules/FindBoost.cmake:2125 (_Boost_MISSING_DEPENDENCIES)
  CMakeLists.txt:45 (find_package)


-- Found Boost: /usr/local/include (found suitable version "1.83.0", minimum required is "1.71.0") found components: unit_test_framework 
-- Performing Test COMPILER_SUPPORTS-Wall
-- Performing Test COMPILER_SUPPORTS-Wall - Success
-- Performing Test COMPILER_SUPPORTS-Wextra
-- Performing Test COMPILER_SUPPORTS-Wextra - Success
-- Performing Test COMPILER_SUPPORTS-Wpedantic
-- Performing Test COMPILER_SUPPORTS-Wpedantic - Success
-- Performing Test COMPILER_SUPPORTS-Wshadow
-- Performing Test COMPILER_SUPPORTS-Wshadow - Success
-- Performing Test COMPILER_SUPPORTS-Wconversion
-- Performing Test COMPILER_SUPPORTS-Wconversion - Success
-- Performing Test COMPILER_SUPPORTS_march_native
-- Performing Test COMPILER_SUPPORTS_march_native - Success
CMake Warning at /usr/local/share/cmake/Modules/FindBoost.cmake:1390 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/local/share/cmake/Modules/FindBoost.cmake:1514 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/local/share/cmake/Modules/FindBoost.cmake:2125 (_Boost_MISSING_DEPENDENCIES)
  CMakeLists.txt:103 (find_package)


CMake Warning at /usr/local/share/cmake/Modules/FindBoost.cmake:1390 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/local/share/cmake/Modules/FindBoost.cmake:1514 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/local/share/cmake/Modules/FindBoost.cmake:2125 (_Boost_MISSING_DEPENDENCIES)
  CMakeLists.txt:103 (find_package)


CMake Warning at /usr/local/share/cmake/Modules/FindBoost.cmake:1390 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/local/share/cmake/Modules/FindBoost.cmake:1514 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/local/share/cmake/Modules/FindBoost.cmake:2125 (_Boost_MISSING_DEPENDENCIES)
  CMakeLists.txt:103 (find_package)


CMake Warning at /usr/local/share/cmake/Modules/FindBoost.cmake:1390 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/local/share/cmake/Modules/FindBoost.cmake:1514 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/local/share/cmake/Modules/FindBoost.cmake:2125 (_Boost_MISSING_DEPENDENCIES)
  CMakeLists.txt:103 (find_package)


CMake Warning at /usr/local/share/cmake/Modules/FindBoost.cmake:1390 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/local/share/cmake/Modules/FindBoost.cmake:1514 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/local/share/cmake/Modules/FindBoost.cmake:2125 (_Boost_MISSING_DEPENDENCIES)
  CMakeLists.txt:103 (find_package)


CMake Warning at /usr/local/share/cmake/Modules/FindBoost.cmake:1390 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/local/share/cmake/Modules/FindBoost.cmake:1514 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/local/share/cmake/Modules/FindBoost.cmake:2125 (_Boost_MISSING_DEPENDENCIES)
  CMakeLists.txt:103 (find_package)


-- Found Boost: /usr/local/include (found suitable version "1.83.0", minimum required is "1.71.0") found components: program_options filesystem system regex timer chrono 
-- Found Eigen3: /usr/local/share/eigen3/cmake
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
-- Found OpenMP_CXX: -fopenmp=libomp (found version "5.0") 
-- Found OpenMP: TRUE (found version "5.0")  
-- Found GROMACS: 2023.2 (suffix: _d)
CMake Warning at CMakeLists.txt:122 (message):
  Gromacs-2020 and above have no support for tabulated interactions, that are
  needed for coarse-graining (see and comment on
  https://gitlab.com/gromacs/gromacs/-/issues/1347)


-- Found Python: /usr/local/bin/python3.9 (found suitable version "3.9.18", minimum required is "3.5") found components: Interpreter 
-- Found SPHINX: /usr/local/bin/sphinx-build (found version "5.3.0") 
-- Could NOT find MKL (missing: MKL_DIR)
-- Found OpenMP_CXX: -fopenmp=libomp (found suitable version "5.0", minimum required is "4.5") 
-- Found OpenMP: TRUE (found suitable version "5.0", minimum required is "4.5")  
-- Found PkgConfig: /usr/local/bin/pkgconf (found version "2.0.3") 
-- Checking for module 'fftw3'
--   Found fftw3, version 3.3.10
-- Found FFTW3: /usr/local/lib/libfftw3.so  
-- Looking for C++ include cassert
-- Looking for C++ include cassert - found
-- Looking for C++ include cmath
-- Looking for C++ include cmath - found
-- Looking for C++ include fstream
-- Looking for C++ include fstream - found
-- Looking for C++ include functional
-- Looking for C++ include functional - found
-- Looking for C++ include iostream
-- Looking for C++ include iostream - found
-- Looking for C++ include limits
-- Looking for C++ include limits - found
-- Looking for C++ include list
-- Looking for C++ include list - found
-- Looking for C++ include map
-- Looking for C++ include map - found
-- Looking for C++ include ostream
-- Looking for C++ include ostream - found
-- Looking for C++ include sstream
-- Looking for C++ include sstream - found
-- Looking for C++ include stack
-- Looking for C++ include stack - found
-- Looking for C++ include stdexcept
-- Looking for C++ include stdexcept - found
-- Looking for C++ include string
-- Looking for C++ include string - found
-- Looking for C++ include vector
-- Looking for C++ include vector - found
-- Found EXPAT: /usr/local/lib/libexpat.so (found version "2.5.0") 
-- Found GMX: /usr/local/bin/gmx_d  
-- Found UnixCommands: /usr/local/bin/bash  
-- Looking for C++ include algorithm
-- Looking for C++ include algorithm - found
-- Looking for C++ include iomanip
-- Looking for C++ include iomanip - found
-- Looking for C++ include numeric
-- Looking for C++ include numeric - found
-- Looking for C++ include cstdlib
-- Looking for C++ include cstdlib - found
-- Found LMP: /usr/local/bin/lmp  
-- Found python module pytest
-- Checking for module 'libint2'
--   Found libint2, version 2.7.2
-- Found libint2: /usr/local/include (found suitable version "2.7.2", minimum required is "2.6") 
-- Found libecpint: /usr/local/lib/cmake/ecpint
CMake Warning at xtp/CMakeLists.txt:27 (message):
  HDF5 will be used such that it is compatible with version 1.8.


-- Found python module lxml
-- Found python module h5py
-- Found python module rdkit
-- Could NOT find Python (missing: Development Development.Module Development.Embed) (found suitable version "3.11", minimum required is "3.5")
-- Found PythonInterp: /usr/local/bin/python3.9 (found suitable version "3.9.18", minimum required is "3.6") 
-- Found PythonLibs: /usr/local/lib/libpython3.9.so
-- Performing Test HAS_FLTO
-- Performing Test HAS_FLTO - Success
-- Performing Test HAS_FLTO_THIN
-- Performing Test HAS_FLTO_THIN - Success
-- Found pybind11: /usr/local/include (found version "2.11.1")
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'ase'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'xmltodict'
-- Found python module pytest
-- Skipping xtp python bindings
-- Found JUPYTER_NBCONVERT: /usr/local/bin/jupyter (found suitable version "7.9.2", minimum required is "5.4.1") 
CMake Error at xtp-tutorials/share/doc/CMakeLists.txt:20 (get_target_property):
  get_target_property() called with non-existent target "VOTCA::xtp_binds".


-- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE) 
-- Could NOT find CLANG_FORMAT: Found unsuitable version "0.0", but required is at least "11" (found CLANG_FORMAT_EXECUTABLE-NOTFOUND)
-- The following features have been enabled:

 * BUILD_XTP, Build xtp module
 * BUILD_TESTING, Enable unit tests
 * INJECT_MARCH_NATIVE, Add -march=native to CXX flags
 * INSTALL_RC_FILES, Install votca rc files (VOTCARC.bash etc.)
 * BUILD_MANPAGES, Build manpages (disable for cross-compile)

-- The following OPTIONAL packages have been found:

 * SPHINX
 * JUPYTER_NBCONVERT (required version >= 5.4.1)

-- The following RECOMMENDED packages have been found:

 * GROMACS (required version >= 2019.6)
   Used to read/write gromacs data files
 * FFTW3
   Enables FFT functionality
 * PkgConfig
   Used to detect fftw package
   Used to detect libint2 package
 * pybind11
   Used to build xtp python bindings

-- The following REQUIRED packages have been found:

 * Boost (required version >= 1.71.0)
   Extended C++ libraries
 * Eigen3 (required version >= 3.3.0)
   C++ vector data structures
 * Threads
   Used for thread parallelization
 * OpenMP (required version >= 4.5)
   Used for thread parallelization in tools
 * EXPAT
   Used for XML parsing
 * UnixCommands
   Used in testing
 * Libxc
   Provides exchange-correclation functionals
 * libint2 (required version >= 2.6)
   Calculates Gaussian integrals
 * ecpint
   Calculates Gaussian integrals over pseudo potentials
 * PythonInterp (required version >= 3.6)
 * PythonLibsNew

-- The following features have been disabled:

 * ENABLE_RPATH_INJECT, Inject rpath into executables
 * BUILD_OWN_GROMACS, Build an internal version of gromacs
 * BUILD_OWN_LIBINT, Build an internal version of libint
 * ENABLE_VALGRIND_TESTING, Run unit tests through valgrind, too
 * INSTALL_CSGAPPS, Install extra app using csg
 * ENABLE_REGRESSION_TESTING, Enable regression tests
 * USE_CUDA, Use CUDA for GPU acceleration

-- The following OPTIONAL packages have not been found:

 * MKL
   Enables accelerated performance with MKL
 * Doxygen
   Needed to build API documentation
 * CLANG_FORMAT (required version >= 11)

-- The following REQUIRED packages have not been found:

 * Python (required version >= 3.5)
   Used in csg scripts and docu
   Used to build xtp python bindings

-- Configuring incomplete, errors occurred!
*** Error code 1
```

FreeBSD 13.2
Python-3.9
clang-16

I am not sure about the cmake python business, but the actual error is above:
```
CMake Error at xtp-tutorials/share/doc/CMakeLists.txt:20 (get_target_property):
  get_target_property() called with non-existent target "VOTCA::xtp_binds".
```
The problem is that, as `ase` and `xmltidict` wasn't found, `pyxtp` was skipped, but we use this in sphinx build unconditionally, which is a bug.

Unless you have a good reason to build documentation you could just disable sphinx with `-DCMAKE_DISABLE_FIND_PACKAGE_SPHINX=ON` and that should work.

--- a/CHANGELOG.rst
+++ b/CHANGELOG.rst
@@ -8,6 +8,7 @@ Version 2023-dev
 -  fix python shebang to python3 (#1093)
 -  Clean-up CI (#1092, #1095)
 -  remove reference to old webpage (#1094)
+-  fix doc generation without pyxtp (#1097)
 
 Version 2023 (released 15.11.23)
 ================================

--- a/xtp-tutorials/CMakeLists.txt
+++ b/xtp-tutorials/CMakeLists.txt
@@ -15,14 +15,18 @@ set(NOTEBOOKS "GROMACS/Methane/QMMM_GROMACS.ipynb")
 list(APPEND NOTEBOOKS "LAMMPS/Thiophene/QMMM_LAMMPS.ipynb")
 list(APPEND NOTEBOOKS "tools/dftgwbse_CH4/DFTGWBSE_ENERGY.ipynb")
 list(APPEND NOTEBOOKS "LAMMPS/KMC_Thiophene/LAMMPS_KMC.ipynb")
-list(APPEND NOTEBOOKS "pyxtp/geometry_optimization.ipynb")
-list(APPEND NOTEBOOKS "pyxtp/single_point.ipynb")
 
 set(NEW_NOTEBOOKS)
 
 get_target_property(XTP_PATH VOTCA::votca_xtp XTP_PATH)
 get_target_property(VOTCASHARE VOTCA::votca_xtp VOTCA_SHARE)
-get_target_property(XTP_PYPATH VOTCA::xtp_binds XTP_PYPATH)
+
+if(TARGET VOTCA::xtp_binds)
+  get_target_property(XTP_PYPATH VOTCA::xtp_binds XTP_PYPATH)
+  list(APPEND NOTEBOOKS "pyxtp/geometry_optimization.ipynb")
+  list(APPEND NOTEBOOKS "pyxtp/single_point.ipynb")
+endif()
+
 foreach(_notebook ${NOTEBOOKS})
   get_filename_component(_FILE "${_notebook}" NAME)
   get_filename_component(_DIR "${_notebook}" DIRECTORY)

